### ISSUE TITLE
SKCamera helper class

### ISSUE THREAD

**Issue by KingSora** [mono/SkiaSharp#214]
I've seen that the original SKIA library has a helper class for the projection transformation matrix called "SkCamera". Please implement this in SkiaSharp too: https://skia.org/user/api/skmatrix

**Comment by mattleibow** [comment]
This was added in commit 39de359bfebdc0b4bdcfe1d19214a4dedfac2286

**Comment by KingSora** [comment]
If I'm trying to initialize a new  SK3dView, I get a "System.EntryPointNotFoundException".

The line of code looks like this:

```
SK3dView v = new SK3dView();
```

I've updated SkiaShap as follows:
1. git submodule init && git submodule update
2. git pull

Then I ran the script:
".\bootstrapper.ps1 -Target externals"
".\bootstrapper.ps1 -Target libs"

Did I miss something or did I something wrong?

Best regards,
Sora
